test/test_suite_test.go

```diff
@@ -39,7 +39,7 @@ import (
 var (
 	log             = logging.DefaultLogger
 	DefaultSettings = map[string]string{
-		"K8S_VERSION": "1.13",
+		"K8S_VERSION": "1.14",
```

Comment:
also you need to change the test/Vagranfile: 

```
$K8S_VERSION = ENV['K8S_VERSION'] || "1.13"
```

Reply:
thanks. Done!